application/tests/unit/common/models/EmailTest.php

```diff
@@ -171,7 +171,8 @@ public function testSend()
 
         $this->assertEquals($initialEmailQueueCount + 1, Email::find()->count(), 'emails in db did not increase by one after saving email');
 
-        $email->send();
+        $n = $email->send();
+        $this->assertEquals(1, $n, 'message not sent');
```

Comment:
If you check, I'm pretty sure it's static. The code works, but it isn't the best.

Reply:
I don't know, and it's like this throughout this repo. I'd rather not risk screwing it up. Thanks for the suggestion, though.